I have a problem with python code, NameError traceback (most recent call last) [closed]/questions/72860940/i-have-a-problem-with-python-code-nameerror-traceback-most-recent-call-last72860940
